synapseclient/table.py

```diff
@@ -1003,48 +1077,46 @@ def _before_synapse_store(self, syn):
-    :param type:                            This field is deprecated. Please use `includeEntityTypes`
-    :param includeEntityTypes:              a list of entity types to include in the view. Supported entity types are:
-
-                                                - EntityViewType.FILE,
-                                                - EntityViewType.PROJECT,
-                                                - EntityViewType.TABLE,
-                                                - EntityViewType.FOLDER,
-                                                - EntityViewType.VIEW,
-                                                - EntityViewType.DOCKER
-
-                                            If none is provided, the view will default to include EntityViewType.FILE.
-    :param addDefaultViewColumns:           If true, adds all default columns (e.g. name, createdOn, modifiedBy etc.)
-                                            Defaults to True.
-                                            The default columns will be added after a call to
-                                            :py:meth:`synapseclient.Synapse.store`.
-    :param addAnnotationColumns:            If true, adds columns for all annotation keys defined across all Entities in
-                                            the EntityViewSchema's scope. Defaults to True.
-                                            The annotation columns will be added after a call to
-                                            :py:meth:`synapseclient.Synapse.store`.
-    :param ignoredAnnotationColumnNames:    A list of strings representing annotation names.
-                                            When addAnnotationColumns is True, the names in this list will not be
-                                            automatically added as columns to the EntityViewSchema if they exist in any
-                                            of the defined scopes.
-    :param properties:                      A map of Synapse properties
-    :param annotations:                     A map of user defined annotations
-    :param local_state:                     Internal use only
-
-    Example::
+    A EntityViewSchema is a [Entity][synapseclient.entity.Entity] that displays all files/projects
+    (depending on user choice) within a given set of scopes.
+
+    Attributes:
+        name:                         The name of the Entity View Table object
+        columns:                      (Optional) A list of [Column][synapseclient.table.Column] objects or their IDs.
+        parent:                       The project in Synapse to which this table belongs
+        scopes:                       A list of Projects/Folders or their ids
+        type:                         This field is deprecated. Please use `includeEntityTypes`
+        includeEntityTypes:           A list of entity types to include in the view. Supported entity types are:
+                                        - EntityViewType.FILE,
```

Comment:
Double check this on the mkdocs page. I've had issues with indentation and properly formatting the unordered lists

Reply:
The includeEntityTypes is formatted as unordered list and with values highlighted in gray.
<img width="693" alt="Screenshot 2024-01-09 at 5 18 48 PM" src="https://github.com/Sage-Bionetworks/synapsePythonClient/assets/90745557/af58d05f-f8b5-4eb0-a71c-b22b2f49c82d">